docs/kyma-environment-broker/03-03-hyperscaler-account-pool.md

```diff
@@ -0,0 +1,25 @@
+# Hyperscaler Account Pool
+​
+To provision clusters through Gardener using the Runtime Provisioner, the Kyma Environment Broker (KEB) requires a hyperscaler (GCP, Azure, AWS, etc.) account/subscription. Managing the available hyperscaler accounts is not in the scope of the KEB. Instead, the available accounts are handled by the Hyperscaler Account Pool (HAP). 
+
+The HAP stores credentials for the hyperscaler accounts that have been set up in advance in Kubernetes Secrets. The credentials are stored separately for each provider and tenant. The content of the credentials Secrets may vary for different use cases. The Secrets are labeled with the **hyperscaler-type** and **tenant-name** labels to manage pools of credentials for use by the provisioning process. This way, the in-use credentials and unassigned credentials available for use are tracked. Only the **hyperscaler-type** label is added during Secret creation, and the **tenant-name** label is added when the account respective for a given Secret is claimed. The Hyperscaler Account Pool is unaware of the Secrets' content.
```

Comment:
The Hyperscaler Account Pool is unaware of the Secrets' content. -> This sounds kinda funky, what do you think about rephrasing this using "opaque"? As in: The contents of the Secrets are opaque to the HAP.